docs/source/api_reference/forecasting.rst

```diff
@@ -469,6 +469,14 @@ Generic framework adapters that expose other frameworks in the ``sktime`` interf
 
     HCrystalBallAdapter
 
+.. currentmodule:: sktime.forecasting.compose
+
+.. autosummary::
+    :toctree: auto_generated/
+    :template: class.rst
+
+    SkforecastAutoreg
```

Comment:
I think the right category is "reduction forecasters".

I would also add a short summary at the top on the three types of reduction forecasters available, and sort them into subcategories now that there are three bunches.

* 1st generation `sktime` native, via `make_reduction`
* 2nd generation `sktime` native, `DirectReductionForecaster` and `RecursiveReductionForecaser`
* from `skforecast`, `SkforecastAutoreg`

Reply:
Sure. Could you please tell what the names of the three subcategories need to be? I don't really get the meaning of 1st/2nd generation.

Reply:
I'll change it